frontend/Themes/Store.js

```diff
@@ -76,6 +76,12 @@ class Store {
   setDefaultThemeName(defaultThemeName: ?string) {
     // Don't accept an invalid themeName as a default.
     this.defaultThemeName = getSafeThemeName(defaultThemeName);
+    // Replace theme with new default if user hasn't select theme
+    const storedThemeKey = getFromLocalStorage(LOCAL_STORAGE_THEME_NAME_KEY);
+    if (storedThemeKey !== CUSTOM_THEME_NAME && !Themes.hasOwnProperty(storedThemeKey)) {
+      this.theme = Themes[this.defaultThemeName];
+      this.themeName = this.defaultThemeName;
+    }
```

Comment:
I'm not sure if I like a method like `setDefaultThemeName` having this kind of side effect. Feels kind of unexpected.

I also wish we had less duplicate code between [these lines](https://github.com/jhen0409/react-devtools/blob/d186b6816be04e202e70b871b2551e1b8a552455/frontend/Themes/Store.js#L46-L50) and the newly-added ones.

Did you consider adding a new method (eg `setDefaultTheme`) that could be shared between the constructor and maybe also invoked for `src/standalone.js`? That way the intent is more clearly expressed and we have less duplication.